dist/wrapper.js

```diff
@@ -2,12 +2,91 @@
 /* @flow */
 'use strict'
 
+/* ::
+import type {BmRequest, Headers} from './wrapper.js'
+*/
+
+const privateVars = new WeakMap()
+
+class BmResponse {
+  constructor (
+    request /* : BmRequest */
+  ) {
+    privateVars.set(this, {
+      headers: {},
+      payload: undefined,
+      request,
```

Comment:
Not using at the moment, but figured it would come in handy down the track. Ill remove and we can add it back if needed.